blog/_posts/2017-10-16-fourth-dotty-milestone-release.md

```diff
@@ -18,7 +18,7 @@ You can learn more about Dotty on our [website](https://dotty.epfl.ch).
 
 <!--more-->
 
-This is our fourth scheduled release according to our [6-week release schedule](https://dotty.epfl.ch/docs/usage/version-numbers.html).
+This is our fourth scheduled release according to our [6-week release schedule](https://dotty.epfl.ch/docs/contributing/procedures/release.html).
```

Comment:
The page version-numbers says its content is outdated: https://github.com/lampepfl/dotty/blob/fa08d004308ff4dee799aafa30939e0647018d72/docs/_docs/usage/version-numbers.md

Reply:
this release procedure is also old, maybe we should just drop the link, as the release procedure from 2017 is not crucial to preserve